Shared/DefaultsKeys.swift

```diff
@@ -34,20 +34,51 @@ extension DefaultsKeys {
     static func addFavoriteFood(_ name: String) {
         if !isFavoriteFood(name) {
             Defaults[\.favoriteFoods].append(name)
+            #if os(iOS)
+            DispatchQueue.global(qos: .background).async {
```

Comment:
Why are you putting this on a background thread? It's not an intensive operation and doesn't require its own thread.